doc/user/content/overview/open-cdc.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Materialize Open CDC"
+description: "The Open CDC format allows Materialize to detect when records are duplicated or out of order."
+menu:
+  main:
+    parent: "overview"
+    weight: 4
+---
+
+
+Change data capture (CDC) tools provide feeds that record any changes to a database. Typically, the feeds are then saved to another platform, like Kafka, for storage or processing.
+
+However, sometimes the stream can have missing or duplicate records, or send records out of order. For example, if Debezium crashes while writing a record, it may retry and write the record again, resulting in a duplicate entry.
+
+To prevent the introduction of this kind of error, Materialize has developed the **Open CDC format**. The Open CDC format carries enough information for a downstream data consumer (like Materialize) to recognize when records are missing, duplicated, or out of order. Each record contains:
+
+- the fields being changed
+- the timestamp for the change
+- the number of messages used to transmit the record
```

Comment:
Is this the `diff` field in the examples below or something else?

Reply:
"Number of messages used to transmit the record" is not quite correct (or at least, not precise enough). The "update" messages contain the data, timestamp, and diff -- For example, a diff of +1 means "insert this record"; a diff of -1 means "delete this record".

The progress messages, as distinct from the update messages, say "for a particular range of timestamps, this is how many update messages we should expect".

Reply:
Agree with @umanwizard here. The "diff" describes the action to apply to the database, introducing or removing as many copies of the presented record as this field indicates.

Reply:
The file has been renamed materialize-cdc.md, but I reworked a lot of the page. You can find this bit under the heading "Update JSON array", but it'd be better to check the entire page.